### PR TITLE
Fixed the content-security-policy

### PR DESCRIPTION
**Pull request by wictorwilen**
Added `teams.microsoft.com` to the Content-Security-Policy so that it works in Chrome

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C3%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201703031433406966-83%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: teams/prerequisites.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ For the UI of your tab - its configuration, content, and (optionally) tab remova
 
 * Make sure that the page can be hosted in an iframe. By default web pages can be iframed by anyone. You may optionally set these headers if you wish to only allow your page to be iframed by Microsoft Teams for extra security:
 	
-	* Set header `Content-Security-Policy: frame-ancestors *.teams.microsoft.com *.skype.com`. Most modern browsers support this.
+	* Set header `Content-Security-Policy: frame-ancestors teams.microsoft.com *.teams.microsoft.com *.skype.com`. Most modern browsers support this.
```
Comment (dfederm) [inline teams/prerequisites.md]: Instead of adding, should we just replace *.teams.microsoft.com with teams.microsoft.com?

Comment (ydogandjiev) [inline teams/prerequisites.md]: We should leave *.teams.microsoft.com in there since our dev environment will be moving to int.teams.microsoft.com soon.

Comment (dfederm) [inline teams/prerequisites.md]: Sounds good

**Pull request merged**